src/js/components/VoterGuide/ItemTinyOpinionsToFollow.jsx

```diff
@@ -123,12 +123,8 @@ export default class ItemTinyOpinionsToFollow extends Component {
             onMouseOver={() => this.onTriggerEnter(org_id)}
             onMouseOut={() => this.onTriggerLeave(org_id)}
             className="card-popover">
-            <div className="card">
-              <div className="card-main">
```

Comment:
this is just an incidental fix. By removing some padding, the popover has a bit more space so the text container is less likely to overlap part of the Follow button.